src/Avalonia.Base/Input/PointerPoint.cs

```diff
@@ -157,6 +157,11 @@ public PointerPointProperties(RawInputModifiers modifiers, PointerUpdateKind kin
                 IsXButton2Pressed = true;
             if (kind == PointerUpdateKind.XButton2Released)
                 IsXButton2Pressed = false;
+
+            if(IsLeftButtonPressed || IsMiddleButtonPressed || IsRightButtonPressed || IsXButton1Pressed || IsXButton2Pressed || IsBarrelButtonPressed)
```

Comment:
This changes the default value of `Pressure` from 0.5 to 1.0 or 0.0 globally. Changing the default shouldn't be part of this PR (but can be discussed in another issue if needed). Please revert that part.